Client/Application/LeanplumIntegration.swift

```diff
@@ -173,6 +181,9 @@ class LeanplumIntegration {
     }
 
     func track(eventName: LeanplumEventName, withParameters parameters: [String: AnyObject]) {
+        guard enabled else {
```

Comment:
`shouldSendToLP` already checks the `enabled` bool. Do you still need to check it here again?

Reply:
`shouldSendToLP` checks, but on the main thread. This is, perhaps, an unnecessary optimization.